org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -10,10 +10,20 @@
     <method name="turnOff"/>
     
     <!-- Get the current ambient temperature -->
-    <method name="getCurrentTemperature">
+    <method name="getAmbientTemperature">
```

Comment:
Oh, the Wink thermostat translator is not merged yet, but it's about to be since I just signed off. [https://github.com/openT2T/translators/pull/18](https://github.com/openT2T/translators/pull/18)

---
In reply to: [68284485](https://github.com/openT2T/translators/pull/22#discussion_r68284485) [](ancestors = 68284485,68283197)